client/src/app/site/motions/models/category-numbering.ts

```diff
@@ -0,0 +1,22 @@
+import { Motion } from '../../../shared/models/motions/motion';
+
+/**
+ * wrapper class for the HTTP Call
+ */
+export class CategoryNumbering {
+    private motions: number[];
```

Comment:
`private motionIds (...)`
since you don't store motions there ;-)

Reply:
Since im sending the complete Object (with the key) and the server expects an object of the form
```
{
  motions: [
    1,
    2,
    3
  ]
}
```
it has to be called "motions"